contracts/Strategy.sol

```diff
@@ -1032,6 +1042,15 @@ contract Strategy is BaseStrategy {
             );
     }
 
+    function ethToWant(uint256 _amtInWei)
+        public
+        view
+        override
+        returns (uint256)
+    {
+        return _amtInWei;
```

Comment:
this should use _fromEth function

Reply:
can't, not enough space in the strat.

Reply:
Comment:

Based on some recommendations here https://soliditydeveloper.com/max-contract-size (see `medium impact`), we would able to reduce the size using fewer internal functions.

I see some internal functions are called from only one place:
- _takeLendingProfit()
- _claimRewards()
- _depositInYVault()
- _depositToAave(uint256 amount)
- _takeVaultProfit()
- _sellAAVEForWant(uint256 _amount)
- _sellInvestmentForWant(uint256 _amount)
- _buyInvestmentTokenWithWant(uint256 _amount)

Then the changes look good to me.

Reply:
removed `_depositInYVault` and I inlined it for a bit more of space.